doc/conf.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+#
+# gem-plotting-tools documentation build configuration file, created by
```

Comment:
`gem-plotting-tools` should bechanged to `gemplotting`, reflecting the actual name of the `python` package